### ISSUE TITLE
the AST of comment, superscript , and subscript

### ISSUE THREAD

**Issue by evpeople** [nvim-neorg/rust-norg#8]
Should we make the `comment`,`superscript `,`subscript` similar to attachModifier? As it stands, for extension development, a significant amount of work is required to filter out the entire content of the comment.

I test on the following content 
```norg

   %This is a comment, and you never see it in your exported output.%
   This is ^superscript^ (it won't render properly in Neovim).
   This is ,subscript,   (same as above).

```
Paragraph([Token(Special('%')), Token(Text("This")), Token(Whitespace), Token(Text("is")), Token(Whitespace), Token(Text("a")), Token(Whitespace), Token(Text("comment")), Token(Special(',')), Token(Whitespace), Token(Text("and")), Token(Whitespace), Token(Text("you")), Token(Whitespace), Token(Text("never")), Token(Whitespace), Token(Text("see")), Token(Whitespace), Token(Text("it")), Token(Whitespace), Token(Text("in")), Token(Whitespace), Token(Text("your")), Token(Whitespace), Token(Text("exported")), Token(Whitespace), Token(Text("output")), Token(Special('.')), Token(Special('%')), Token(Whitespace), Token(Text("This")), Token(Whitespace), Token(Text("is")), Token(Whitespace), Token(Special('^')), Token(Text("superscript")), Token(Special('^')), Token(Whitespace), Token(Special('(')), Token(Text("it")), Token(Whitespace), Token(Text("won")), Token(Special('\'')), Token(Text("t")), Token(Whitespace), Token(Text("render")), Token(Whitespace), Token(Text("properly")), Token(Whitespace), Token(Text("in")), Token(Whitespace), Token(Text("Neovim")), Token(Special(')')), Token(Special('.')), Token(Whitespace), Token(Text("This")), Token(Whitespace), Token(Text("is")), Token(Whitespace), Token(Special(',')), Token(Text("subscript")), Token(Special(',')), Token(Whitespace), Token(Special('(')), Token(Text("same")), Token(Whitespace), Token(Text("as")), Token(Whitespace), Token(Text("above")), Token(Special(')')), Token(Special('.'))])
```

**Comment by evpeople** [comment]
```norg
This is a spoiler: !org-mode is a bozo!, to see the content you either enter insert mode on this line (good luck) or toggle the concealer off.
```
The spoiler has the same problem or feature

**Comment by benlubas** [comment]
Yeah, these should all behave like attached mods, b/c they are attached mods
